lib/lita/handlers/reviewme.rb

```diff
@@ -120,8 +122,14 @@ def next_reviewer(room)
         ns_redis(room.id).rpoplpush(REDIS_LIST, REDIS_LIST)
       end
 
-      def github_comment(reviewer)
-        ":eyes: @#{reviewer}"
+      def github_comment(reviewer, pull_request)
+        msg = config.github_comment_template || DEFAULT_GITHUB_MSG
+
+        if msg.respond_to?(:call) # its a proc
```

Comment:
gah, i sort of hate to have a condition here, but i can't think of a more elegant way to handle it atm. works! 💃